modules/fts/pages/fts-accessing-managing-full-text-indexes.adoc

```diff
@@ -0,0 +1,23 @@
+= Accessing and Managing Full Text Indexes
+
+[abstract]
+Full Text indexes, which are available under the *Search* tab of the Couchbase Web Console, are another form of the index provided by the Search service.
+
+Full Text indexes are special-purpose indexes that contain targets derived from the textual contents of the documents within one or more buckets or collections from the buckets.
```

Comment:
Too much repetition of the word "Full Text indexes" in this section alone.

This needs refactoring to something like this ..

These indexes analyze and store tokens from text, numeric, geo, boolean, datetime fields within JSON documents obtained from couchbase buckets/collections.